### ISSUE TITLE
Some hypothesis patterns are too generic

### ISSUE THREAD

**Issue by Aremaki** [aphp/edsnlp#154]
Some hypothesis patterns are too generic

Specifically, in the *following patterns*, the terms ":\n" and ": \n" are too generic and for instance:

"the patient with covid receives the following care: 
- ventilation
- etc."

```python
import spacy

nlp = spacy.blank("fr")
nlp.add_pipe("eds.hypothesis")
text = "patient atteint de covid recoit les soins suivant :\n- ventilation\n- etc."
doc = nlp(text)
doc.ents[-1]._.hypothesis
True
```

**Comment by percevalw** [comment]
Indeed! As discussed, these rules could either be disabled by default or improved to detect false positives such as those mentioned in your issue
